components/dashboard/visualizer.go

```diff
@@ -64,8 +64,8 @@ func runVisualizer(component *app.Component) {
 				// 	currentSlot.Store(int64(block.ID().Index()))
 				// }
 			}, event.WithWorkerPool(component.WorkerPool)).Unhook,
-			deps.Protocol.Events.Engine.BlockGadget.BlockAccepted.Hook(func(block *blocks.Block) {
-				sendVertex(block, block.IsAccepted())
+			deps.Protocol.Events.Engine.BlockGadget.BlockPreAccepted.Hook(func(block *blocks.Block) {
+				sendVertex(block, block.IsPreAccepted())
```

Comment:
Use accepted instead? I think it would make more sense as that flag actually advances the tangle time.